dbms/src/Operators/MergeSortBaseTransformOp.cpp

```diff
@@ -143,7 +151,10 @@ OperatorStatus LocalSortTransformOp::transformImpl(Block & block)
 
         // execute partial sort and store the sorted block in `sorted_blocks`.
         SortHelper::removeConstantsFromBlock(block);
-        sortBlock(block, order_desc, limit);
+        if constexpr (do_partial_sort)
```

Comment:
I wander if we change the impl of `executeLocalSort` so that in `executeLocalSort`, it first adds `PartialSortTransformOp`, then adds `MergeSortBaseTransformOp<false>`, then there is no need to make `MergeSortBaseTransformOp` a template class?

Reply:
ok, `LocalSortTransform` has been removed, and execute `PartialSortTransformOp --> MergeSortTransformOp` as local sort.